openedx/features/job_board/models.py

```diff
@@ -0,0 +1,22 @@
+from django.db import models
+from django_countries.fields import CountryField
+from model_utils.models import TimeStampedModel
+
+from .constants import JOB_COMPENSATION_CHOICES, JOB_HOURS_CHOICES, JOB_TYPE_CHOICES
+
+
+class Job(TimeStampedModel):
+    """
+    This model contains all the fields related to a job being
+    posted on the job board.
+    """
+    title = models.CharField(max_length=255, null=False, blank=False)
+    company = models.CharField(max_length=255, null=False, blank=False)
+    type = models.CharField(max_length=32, choices=JOB_TYPE_CHOICES, blank=False)
+    compensation = models.CharField(max_length=32, choices=JOB_COMPENSATION_CHOICES, blank=False)
+    hours = models.CharField(max_length=32, choices=JOB_HOURS_CHOICES, blank=False)
```

Comment:
How about using max_length=255?

Reply:
why 255 @zeshanarifios if we can specify?

Reply:
As this is the max no of characters a CharField can store, And suppose if current data has max 32 characters and we specify max_length=255, This field doesn't take more memory than 32 characters. But in future if we want our field to store characters more than 32, we don't need to update our code and make new migrations.

Reply:
if it doesn't take any more memory why they have given us this option? Just asking I'm curious.

Reply:
This option is for when you want to enforce max_length. Means value **must** not be greater than max_length.

Reply:
CharField creates a VARCHAR field in database that is a variable length field.

<img width="975" alt="Screen Shot 2020-03-27 at 11 41 04 PM" src="https://user-images.githubusercontent.com/26452654/77789070-2fe8c780-7084-11ea-88e1-259d296d3946.png">